pkgs/development/python-modules/django_tagging/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchurl, django }:
+
+buildPythonPackage rec {
+  name = "django-tagging-0.4.5";
+
+  src = fetchurl {
```

Comment:
I think for now, to make it easy for me, just moving `buildPythonPackges` out and use `callPackage` first.